.github/workflows/test_security.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   integ-test-with-security-linux:
     strategy:
       matrix:
-        java: [11, 17, 21]
+        java: [11, 17,  21.0.1]
```

Comment:
Yay, security tests pass now!

Reply:
I've applied this change in #429 so you can just rebase to main and leave it unchanged

Reply:
PR https://github.com/opensearch-project/flow-framework/pull/432 fixes our multi-node integration tests, perhaps lets merge that PR in first prior to this PR so we can gauge if these changes have any effect.